### ISSUE TITLE
Adopting JSON API v1 spec

### ISSUE THREAD

**Issue by jonnedeprez** [apiaryio/mson#46]
Are there any plans for improved support for the JSON API spec? It would be awesome if I could simply specify a resource by its attributes and then get a generated JSONAPI v1 compliant schema for the common REST operations from Apiary.

**Comment by zdne** [comment]
@jonnedeprez Yes! There is a plan for it. However this is more in the jurisdiction of API Blueprint itself as it needs the information about available affordances (`links` in JSON API). See https://github.com/apiaryio/api-blueprint/issues/291

Please feel free to comment, bump or suggest improvements to the idea! (Or even implement it :smile: )
